certified-connectors/Eden AI/Readme.md

```diff
@@ -0,0 +1,169 @@
+
+# Eden AI

```

Comment:
@queSaDiLLaSS this is a great readme file. May I ask you to clarify the name of the connector? in the readme you have it as "Eden AI" and in the swagger you have "title": "Launch". please make sure that the name matches in all files.
Also may I ask to include all required sections in the readme. Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).